components/sock_utils/include/netdb_macros.h

```diff
@@ -5,6 +5,16 @@
  */
 #pragma once
 
+#ifndef CONFIG_IDF_TARGET_LINUX
```

Comment:
Do we need to `#include <sdkconfig.h>` before this check?

Reply:
yes, we do! Thanks for catching this!